### PR TITLE
Add support for ruby 2.6

### PR DESCRIPTION
**Pull request by matheusvetor**

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
Thanks @matheusvetor. Since #479 arrived first, I say let's wait for that PR to land, and then use this PR to add ruby 2.6 to our CI. Sounds good?

**Comment by matheusvetor** [comment]
Sounds good, lets check the progress on #479. Until them, this PR covers all the steps to add another ruby version test support?

**Comment by deivid-rodriguez** [comment]
Yeah, absolutely. Only thing needed as far as I recall is to add one new entry per supported Rails to the CircleCI config that uses the new ruby as the base image.

**Comment by deivid-rodriguez** [comment]
You can now go ahead and rebase this PR @matheusvetor!

**Review by deivid-rodriguez — CHANGES_REQUESTED** [review]
@matheusvetor Can you undo the changes to the gemfiles, and limit this PR to adding entries to test the current set of gemfiles against ruby 2.6? Thanks!

**Comment by matheusvetor** [comment]
@deivid-rodriguez I just added missing jruby test against rails 6beta

File: .circleci/config.yml [hunk 1]
```diff
@@ -133,6 +133,56 @@ jobs:
...
 
+  ruby25rails60beta:
+    docker:
+    - image: circleci/ruby:2.5.1
+
+    environment:
+      BUNDLE_GEMFILE: test/gemfiles/Gemfile-Rails-6-0-beta
+
+    steps:
+      *test_steps
```
Comment (deivid-rodriguez) [inline .circleci/config.yml:144]: This job is already in master. Can you make sure this PR is properly rebased?

**Review by deivid-rodriguez — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (matheusvetor) [reply .circleci/config.yml:144]: I alread rebased after the previous PR was merged.

Comment (deivid-rodriguez) [reply .circleci/config.yml:144]: I see that now, Github is showing the merged commit in #479 as part of this PR. I'm :astonished:.

Comment (deivid-rodriguez) [reply .circleci/config.yml:144]: Can you try squashing commits to see if it fixes the diff? Feel free to leave the addition of the forgotten jruby + 2.5 entry as a separate commit.

Comment (matheusvetor) [reply .circleci/config.yml:144]: A easy way is to cancel this PR and open another with the correct way

Comment (deivid-rodriguez) [reply .circleci/config.yml:144]: Yeah, no sure what happened here... The other PR looks great! :+1:

**Pull request closed without merging**